sonic-slave-bullseye/Dockerfile.j2

```diff
@@ -515,3 +515,12 @@ RUN apt-get install -y nodejs
 # Tell azure pipeline to use node.js in the docker
 LABEL "com.azure.dev.pipelines.agent.handler.node.path"="/usr/bin/node"
 {% endif -%}
+
+# Install Bazel build system (amd64 and arm64 architectures are supported using this method)
+# TODO(PINS): Remove once pre-build Bazel binaries are available for armhf (armv7l)
+{%- if CONFIGURED_ARCH == "amd64" or CONFIGURED_ARCH == "arm64" %}
+ARG bazelisk_url=https://github.com/bazelbuild/bazelisk/releases/latest/download/bazelisk-linux-{{ CONFIGURED_ARCH }}
+RUN curl -fsSL -o /usr/local/bin/bazel ${bazelisk_url} && chmod 755 /usr/local/bin/bazel
+# Bazel depends on "python" command, but python3 is okay
+RUN ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
Why this line is only needed in sonic-slave-bullseye? Is it possible to make bazel help without global change?

Reply:
Before bullseye, the `python` executable exists. So this is only needed for bullseye.

Reply:
We can try to see if that can be a fixed with a bazel option, but I'm not sure

Reply:
try install python3-is-python package?

Reply:
Still prefer change locally for bazel only, no global impact.

Reply:
Unfortunately, I think `python` is hardcoded in to the rule that creates the Debian package for P4RT, so the best solution is to have that somewhere on the path. We should be able to revisit this in the future and remove the `python3-is-python` package.

Reply:
Package has basically no overhead:

```bash
root@4f5472f7998b:/# apt install python-is-python3
....
After this operation, 0 B of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://deb.debian.org/debian bullseye/main amd64 python-is-python3 all 3.9.2-1 [2800 B]
Fetched 2800 B in 0s (52.4 kB/s)            
...
Setting up python-is-python3 (3.9.2-1) ...
root@4f5472f7998b:/# dpkg -L python-is-python3
/.
/usr
/usr/bin
/usr/share
/usr/share/doc
/usr/share/doc/python-is-python3
/usr/share/doc/python-is-python3/changelog.gz
/usr/share/doc/python-is-python3/copyright
/usr/share/man
/usr/share/man/man1
/usr/bin/python
/usr/share/man/man1/python.1.gz
root@4f5472f7998b:/# ls -l /usr/bin/python
lrwxrwxrwx 1 root root 7 Mar  2  2021 /usr/bin/python -> python3
```